src/components/App/index.tsx

```diff
@@ -104,6 +105,8 @@ export default class App extends Component<Props, State> {
     ],
   };
 
+  private snackbar?: SnackBarElement = undefined;
```

Comment:
You don't need the `= undefined` fwiw.

Reply:
without it, typescript complains that the property is uninitialized. It has to be defined either here or in the constructor, and there's no useful value to set it to in the constructor